Components/GameBoard/Messages.jsx

```diff
@@ -116,16 +116,20 @@ class Messages extends React.Component {
                     </span>
                 );
             } else if(fragment.name && fragment.argType === 'player') {
+                let userClass = 'username' + (fragment.role ? ` ${fragment.role}` : '');
```

Comment:
I might suggest namespacing the CSS classes more precisely, e.g. `role-admin`, `role-contributor`, etc, to avoid any potential future name collisions.

It might also be nice if role was always *some* value, instead of sometimes being `undefined`, but not a big deal either way.

Reply:
I can change it but it will be undefined for a while until people login again and get a new value cached on their user object